php/tests/WellKnownTest.php

```diff
@@ -416,4 +416,47 @@ public function enumNameValueConversionDataProvider()
             ['\Google\Protobuf\Syntax'],
         ];
     }
+
+    /**
+     * @dataProvider optionalFieldsDataProvider
+     */
+    public function testReflectionProperty($property, $default)
+    {
+        $testMessage = new TestMessage();
+
+        $functionName = preg_replace_callback('/[_\-\s]+(.)?/',
+            function ($matches) {return strtoupper($matches[1] ?? '');},
```

Comment:
```suggestion
            fn ($matches) => strtoupper($matches[1] ?? ''),
```

Or alternatively (easier to read, and we do it elsewhere in this library):

```php
$functionName = implode('', array_map('ucwords', explode('_', $property)));
```

Reply:
The `ucwords` version actually looks better than the `preg_match` version.